build.gradle

```diff
@@ -23,6 +23,21 @@ repositories {
     mavenCentral()
 }
 
+bootRun {
+    environment dotenv()
+}
+
+def dotenv() {
```

Comment:
Remove this dotenv() function and replace it with a dependency `https://mvnrepository.com/artifact/me.paulschwarz/spring-dotenv/4.0.0`